dist/config/packages/monsieurbiz_sylius_rich_editor_plugin_custom.yaml

```diff
@@ -10,5 +10,3 @@ monsieurbiz_sylius_richeditor:
             templates:
```

Comment:
It is but at the end, we will not have recipes files anymore when we will have the recipes on Symfony side.

Reply:
It's not a recipe file but a custom we want to add in our application

Reply:
See my comment https://github.com/monsieurbiz/SyliusRichEditorPlugin/pull/168#discussion_r741106130 about the use of symlinks

I think dist files will be managed differently in the future but it's a first step to avoid to manage a full test application